docs/quickstart.md

```diff
@@ -0,0 +1,111 @@
+## Quickstart
```

Comment:
I don't really like that this doesn't clarify that this is a quickstart for running the agent straight on a host (and not e.g. in docker or on k8s).  Can you add some text to clarify that?